app/models/setting/rh_cloud.rb

```diff
@@ -7,6 +7,7 @@ def self.load_defaults
       [
         set('allow_auto_inventory_upload', N_('Allow automatic upload of the host inventory to the Red Hat cloud'), true),
         set('obfuscate_inventory_hostnames', N_('Obfuscate host names sent to Red Hat cloud'), false),
+        set('rh_cloud_token', N_('Authentication token to Red Hat cloud services. Used to authenticate requests to cloud APIs'), 'DEFAULT', N_('Red Hat Cloud token'), nil, encrypted: true),
```

Comment:
with 1.20 the initial setting doesn't get encrypted as it was in 1.24 and master:

![Selection_334](https://user-images.githubusercontent.com/26363699/85393592-eeb66580-b555-11ea-94e1-513b2ba378e9.png)